core/src/com/mygdx/game/core/ecs/component/Coordinates.java

```diff
@@ -0,0 +1,29 @@
+package com.mygdx.game.core.ecs.component;
+
+import com.artemis.PooledComponent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Coordinates extends PooledComponent {
+  private int x;
```

Comment:
To nie ok. Wszystko co zmieniło Position na Coordinates to jest do wywalenia. Chciałem inaczej napisać kod serwerowy, żeby nie wiedział konkretnej pozycji w postaci float tylko współrzędne i żeby to client sam sobie decydował jak to rozplasować, ale się nie udało jakoś.